gen_rust.py

```diff
@@ -406,6 +406,11 @@ def write_exc(filename, action):
 
     ### dnn ###
     "cv::dnn::_Range",
+
+    ### xfeatures2d ###
+    # Requires opencv to be configured with -DOPENCV_ENABLE_NONFREE=ON
+    # and needs some gen_rust.py changes to build correctly.
+    "cv::xfeatures2d",
```

Comment:
I happen to have opencv compiled with `-DOPENCV_ENABLE_NONFREE=ON`, which causes this new header to appear which isn't currently in the committed rs files. It actually compiled fine on my machine, but travis doesn't like it. In any case, I think adding those new classes should be a separate PR.

Reply:
moved to https://github.com/twistedfall/opencv-rust/pull/37/files#diff-a7b0a2dee0126cddf994326e705a91eaR121